gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java

```diff
@@ -197,15 +197,15 @@ private static LatLng toLatLng(Point point) {
   }
 
   private void removeMarker(Marker marker) {
-    Log.v(TAG, "Removing marker " + marker.getId());
+    Timber.tag(TAG).v("Removing marker %s", marker.getId());
```

Comment:
We don't need to mention a custom tag. Timber uses the filename as the default tag. 

So, the above can be written as `Timber.v("Removing marker %s", marker.getId());`

Reply:
This also makes `TAG` variable unused. So, please remove that as well

Reply:
Gotcha, done! 513ef9a